code/datums/shuttles.dm

```diff
@@ -124,6 +124,13 @@
 	port_id = "mining"
 	can_be_bought = FALSE
 
+/datum/map_template/shuttle/mining_common
+	port_id = "mining_common"
+	can_be_bought = FALSE
+
+/datum/map_template/shuttle/mining_common/meta
+	suffix = "meta"
```

Comment:
This is the base shuttle types section, move the actual shuttle to the section below with the rest of the actual shuttles